### PR TITLE
Add WASM doom game example in Docker Compose file

### PR DESCRIPTION
**Pull request by cedricclyburn**
It is fun to demo :)

### CODE REVIEW

**Comment by slemeur** [comment]
This is great @cedricclyburn ! Maybe, we should have a small readme to explain how to run it?

**Comment by cedricclyburn** [comment]
Sure thing! Added @slemeur

**Review by benoitf — APPROVED** [review]

File: wasm/doom/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## Podman Compose
+
+To run the game, you need to have [Podman](https://podman.io/) installed, and the [`podman-compose`](https://github.com/containers/podman-compose) script available (included in Podman Desktop).
+
+### Instructions
+
+When in this directory, run the `podman-compose` command to bring up the container:
+
+```bash
+podman-compose up
```
Comment (cdrage) [inline wasm/doom/README.md:14]:
```suggestion
podman compose up
```

File: wasm/doom/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# WASM Port of Doom game in browser
+
+This is a port of the [Doom game](https://en.wikipedia.org/wiki/Doom_(1993_video_game)) to the web using [WebAssembly](https://webassembly.org/).
+
+## Podman Compose
+
+To run the game, you need to have [Podman](https://podman.io/) installed, and the [`podman-compose`](https://github.com/containers/podman-compose) script available (included in Podman Desktop).
```
Comment (cdrage) [inline wasm/doom/README.md:7]: should not link to `podman-compose` as we are prefering `podman compose` now using the compose binary

**Review by cdrage — CHANGES_REQUESTED** [review]

**Comment by cedricclyburn** [comment]
Awesome should be good to go!

**Review by slemeur — APPROVED** [review]

**Comment by slemeur** [comment]
Thanks @cedricclyburn !!

**Pull request merged**